components/tyk-pump/values.yaml

```diff
@@ -171,7 +171,7 @@ pump:
   containerPort: 9090
 
   # backend defines the pumps to be created by default, as an array of string.
-  # Supported backends are ["mongo", "postgres", "prometheus","hybrid"]
+  # Supported backends are ["mongo", "postgres", "postgres-aggregate", "prometheus","hybrid"]
```

Comment:
One minor request is that even though `postgres-aggregate` looks fine to me, we refer to it as `sql-aggregate` in the docs: https://tyk.io/docs/tyk-pump/tyk-pump-configuration/tyk-pump-dashboard-config/#2-sql-aggregate-pump

Can you please update it to `sql-aggregate` to make the naming compatible with docs?

Reply:
By your logic the already existing option "postgres" should be called "sql".

I can change it if you really want to, but then the naming will be inconsistent.
There should either be "postgress" and "postgress-aggregate" or "sql" and "sql-aggregate".

Reply:
Also the implementation is for postgress specifically, not sql, e.g. 
```
"tyk-pump.pg_connection_string_secret_name"
```

Reply:
makes sense! then, let's keep as it is.